build/templates/session.py.mako

```diff
@@ -245,7 +219,7 @@ context_name = 'acquisition' if c['direction'] == 'input' else 'generation'
         error_code = self.library.${c_function_prefix}${func_name}(${helper.get_library_call_parameter_snippet(f['parameters'])})
         # Don't use _handle_error, because positive value in error_code means size, not warning.
         if (errors._is_error(error_code)):
-            raise errors.Error(self.library, self.vi, error_code)
+            raise errors.Error(self, error_code)
```

Comment:
Wasn't there an issue for this, or didn't @Fladolcetta fix this separately?

Reply:
He had a fix, but since I needed to fix it as part of fixing error handling in general, I said I would put the fix in here.

I don't see an open issue for it though